.github/workflows/tests.yml

```diff
@@ -94,13 +94,13 @@ jobs:
         run: find $PWD -type f \( -name 'WORKSPACE' -o -name '*.bzl' -o -name '*.bazel' \) | xargs bazel run buildifier -- -lint=fix && git diff --exit-code
       - name: Check docs
         run: bazelisk run docs && git diff --exit-code docs
-  xcodeproj_tests_xcode_12_2:
-    name: .xcodeproj Tests on Xcode 12.2
+  xcodeproj_tests_xcode_12_5_1:
+    name: .xcodeproj Tests on Xcode 12.5.1
     runs-on: macos-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Select Xcode 12.2
-        run: sudo xcode-select -s /Applications/Xcode_12.2.app
+      - name: Select Xcode 12.5.1
+        run: sudo xcode-select -s /Applications/Xcode_12.5.1.app
```

Comment:
It looks like this one got missed - consider to just update it to use `.xcode_select.sh`